core-test/src/test/scala/org/bitcoins/core/protocol/Bech32mSpec.scala

```diff
@@ -0,0 +1,82 @@
+package org.bitcoins.core.protocol
+
+import org.bitcoins.core.protocol.script.WitnessVersion0
+import org.bitcoins.core.util.{Bech32, Bech32Encoding}
+import org.bitcoins.testkit.core.gen._
+import org.scalacheck.{Prop, Properties}
+
+import scala.annotation.tailrec
+import scala.util.{Random, Success}
+
+class Bech32mSpec extends Properties("Bech32mSpec") {
```

Comment:
nit: This is the old style of test cases (uses scalacheck, not scalatest). We are trying to slowly move away from these over the long term. Can you write these using `BitcoinSUnitTest` ? Also convert the `Bech32Spec` file if you are feeling ambitious :-)

Reply:
https://github.com/bitcoin-s/bitcoin-s/blob/c5be81d5e21cd22a4e69dc86dcf47fbaa7d623f3/core-test/src/test/scala/org/bitcoins/core/protocol/AddressTest.scala#L12

Reply:
I guess this means this file can be deleted and test cases migrated to `Bech32mTest`